### ISSUE TITLE
Release 1.1

### ISSUE THREAD

**Issue by PhilippSelenium** [bittner/pyclean#18]
Is it possible to release a new version in order to get all the fixes from the master branch?
Version 1.0 is just broken on python3.6
```
$ pyclean .
pyclean.py", line 63
    except (IOError, OSError), e:
                             ^
SyntaxError: invalid syntax
```

**Comment by bittner** [comment]
Try using the explicit alias `py3clean`, as I explained in #17. I hope that helps.

The current `master` mostly contains preparations for an upcoming 2.0.0 release. It's seems broken, actually, for the new `--legacy` option due to the fact that there were never tests for the Debian scripts that I integrated.

I'll try to work on the next release in the upcoming weeks.

**Comment by PhilippSelenium** [comment]
@bittner Thanks py3clean works.

**Comment by bittner** [comment]
I have released v2.0.0, which should now work on any platform supported by Python3.

Can you try whether it does what you expect?

```console
python3 -m pip install --upgrade pyclean
```
```console
pyclean --version
2.0.0
```
```console
pyclean --help
```

**Comment by PhilippSelenium** [comment]
@bittner Thank you. Works fine for me!
